roles/openshift_aws/defaults/main.yml

```diff
@@ -220,7 +222,7 @@ openshift_aws_ami_map:
   infra: "{{ openshift_aws_ami }}"
   compute: "{{ openshift_aws_ami }}"
 
-openshift_aws_master_group:
```

Comment:
There is no reason to combine this with openshift_aws_node_groups.  The values never traverse the same paths.